applications/portal/backend/api/management/ingestion/users_ingestor.py

```diff
@@ -45,7 +45,7 @@ def _create_keycloak_user_from_row(self, *args):
             "requiredActions": [KeycloakRequiredActions.UPDATE_PASSWORD.value]
         }, exist_ok=True)
         if row['orcid_id'] != '':
-            self.keycloak_admin.add_user_social_login(row['email'], PROVIDER_ID, row['orcid_id'], row['orcid_id'])
+            self.keycloak_admin.add_user_social_login(user, PROVIDER_ID, row['orcid_id'], row['orcid_id'])
```

Comment:
does this api accept a user object? https://github.com/marcospereirampj/python-keycloak/blob/master/src/keycloak/keycloak_admin.py#L923

Reply:
`user` is not actually an user object but an id. I renamed the variable to make it clear.
![image](https://user-images.githubusercontent.com/19196034/211785431-bd5befa8-8f8b-4617-be6c-c6ed597cb406.png)

Reply:
I also re-tested it and it is indeed creating the orcid connection:
![image](https://user-images.githubusercontent.com/19196034/211785861-3d093447-cc09-44ce-a38b-567ccc82c3ee.png)